test/e2e/node_test.go

```diff
@@ -274,7 +274,6 @@ var _ = Describe("When managing identities from the underlying nodes", func() {
 
 		By(fmt.Sprintf("Ensuring both keyvault-identity and cluster-identity are assigned to %s", node.Name))
 		userAssignedIdentities := azureClient.ListUserAssignedIdentities(node.Spec.ProviderID)
-		Expect(len(userAssignedIdentities) == 2).To(BeTrue())
```

Comment:
Removing this line because clusters can have external user-assigned identities that are not part of the e2e test suite.